crates/bazed-core/src/app.rs

```diff
@@ -0,0 +1,80 @@
+use std::sync::Arc;
+
+use bazed_rpc::{core_proto::ToBackend, core_proto::ToFrontend, server::ClientSendHandle};
+use color_eyre::Result;
+use futures::StreamExt;
+use tokio::sync::RwLock;
+
+use crate::{document::Document, marked_rope::MarkerId};
+
+pub struct App {
+    document: Option<Document>,
+    event_send: ClientSendHandle,
+    cursors: Vec<MarkerId>,
+}
```

Comment:
Please find a more descriptive name for your internal state struct.

Reply:
I disagree with "App is the perfect name for the inter-
I also disagree with convention.